scripts/analysis_1/nbconverted/3_visualize_data.py

```diff
@@ -287,12 +310,13 @@
 combined_data_df = pd.concat([input_data_PCAencoded_df, simulated_data_PCAencoded_df])
 
 # Plot
-ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='color_by'), alpha=0.5) + facet_wrap('~dataset') + xlab('PCA 1') + ylab('PCA 2') + ggtitle('PCA of original and simulated data (gene space)')
+ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='experiment_id'), alpha=0.3) + facet_wrap('~dataset') + xlab('PCA 1') + ylab('PCA 2') + ggtitle('PCA of original and simulated data (gene space)')
```

Comment:
I think it's a good attempt. The plot is not very informative though. Human eyes are useless noticing subtle differences between ~40 colors at a=0.3. Consider grouping them based on growth_condition(1,2), strain, or something that represents some level of expected biological similarity between experiments.

Reply:
You're right.  

Previously, I plotted using only 5 experiments in order to examine if the experiment structure was preserved by the simulation -- and it was.  Then I plotted using more experiments and I was more looking to see that the distribution of points was conserved between the original and simulated, but I couldn't say anything about any particular experiment.

You raise a good point though, about how to show/validate that our VAE simulation is generating gene expression data that is representative of real data.  And one way we can try to do this is by coloring by strain or growth condition and comparing the clustering